src/main/java/com/fabriccommunity/thehallow/world/biome/GhastlyDesert.java

```diff
@@ -14,7 +14,7 @@
 
 public class GhastlyDesert extends HallowedBaseBiome {
 	public GhastlyDesert() {
-		super(new Settings().surfaceBuilder(DESERT_SURFACE_BUILDER).precipitation(Precipitation.NONE).category(Category.DESERT).precipitation(Biome.Precipitation.NONE).category(Biome.Category.DESERT).depth(0.125F).scale(0.05F).temperature(2.0F).downfall(0.0F).waterColor(0xBB0A1E).waterFogColor(0xBB0A1E));
+		super(new Settings().surfaceBuilder(DESERT_SURFACE_BUILDER).precipitation(Precipitation.NONE).category(Category.DESERT).precipitation(Biome.Precipitation.NONE).category(Biome.Category.DESERT).depth(0.125F).scale(0.05F).temperature(2.0F).downfall(0.0F).waterColor(4159204).waterFogColor(329011));
```

Comment:
Why change these to decimal ints instead of hex? Seems like it'd decrease readability...

(this goes for all affected biome classes)

Reply:
All I did was simply copy over the color codes from the Plains biome, which was already in integer form. I'll change them now.